### DISCUSSION TITLE
storage/hwameistor/intro/ls/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1678] [Announcements]
# storage/hwameistor/intro/ls/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/storage/hwameistor/intro/ls/

**Comment by liyuerich** [comment]
现在是用hwameistor operator 安装，是否可以介绍一下？
